### PR TITLE
feat: Edit project admin from the Roles edit form

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1500

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

Go to https://deploy-preview-2054--regen-marketplace.netlify.app/ecocredits/projects, logging in with an address that is the admin of some project, and edit the project admin from the roles form

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie see testing instructions above

File: web-marketplace/src/components/organisms/RolesForm/components/AdminModal/AdminModal.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { useFormState, useWatch } from 'react-hook-form';
+
+// import { useStyles } from './AdminModal.styles';
```
Comment (flagrede) [inline web-marketplace/src/components/organisms/RolesForm/components/AdminModal/AdminModal.tsx:3]: To remove ?

**Review by flagrede — APPROVED** [review]
tACK

**Comment by erikalogie** [comment]
@blushi I'm having trouble testing this because I'm getting an error when trying to log in. I can log in just fine on dev.app.regen.network

Screen Shot 2023-08-30 at 5 48 50 AM

Screen Shot 2023-08-30 at 5 50 17 AM

Any idea what is happening here?

**Comment by blushi** [comment]
> Any idea what is happening here?

 This used to work for me yesterday but got this same error once, then it worked back again, this error is coming from keplr code, not sure why... I've triggered a new build, could you reload the page and try again now?

**Comment by blushi** [comment]
@erikalogie could you have another try? This PR intersects a bit with my current work on https://github.com/regen-network/regen-web/issues/2039 so I'd like to get it merge ASAP so I don't have to deal to conflicts and I can fully implement the logic for admin edit too there.

**Comment by erikalogie** [comment]
Ok, it is working again for me.

One thing I noticed is that we are missing this paragraph in the popup: https://monosnap.com/file/ArZlDYu0FKiSMN4xaj2MwtZog0qYwl

Also, can we display the new project admin and the old project admin in the confirmation popup?
https://monosnap.com/file/FRRbsQjPshLKwpcrLq7LJAahdxiIoq

**Comment by blushi** [comment]
> Ok, it is working again for me.
> 
> One thing I noticed is that we are missing this paragraph in the popup: https://monosnap.com/file/ArZlDYu0FKiSMN4xaj2MwtZog0qYwl
> 
> Also, can we display the new project admin and the old project admin in the confirmation popup? https://monosnap.com/file/FRRbsQjPshLKwpcrLq7LJAahdxiIoq

Added! Please wait a few minutes for the new build to finish before testing again.

**Pull request merged**